CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.13.6
+
+_Jun 21, 2023_
+
+A big thanks to the 25 contributors who made this release possible. Here are some highlights ✨:
```

Comment:
We should add highlights here, you can check previous examples, I would propose including the new example using next.js's app directory.